src/mmw/apps/home/views.py

```diff
@@ -29,5 +30,22 @@ def home_page(request):
     return render_to_response('home/home.html', csrf_token)
 
 
+def model(request, proj_id=None):
+    """
+    If proj_id was specified, check that the user owns
+    the project. It not, return a 404. Otherwise, just
+    load the index template and the let the front-end
+    handle the route and request the project through
+    the API.
+    """
+    csrf_token = {}
+    csrf_token.update(csrf(request))
```

Comment:
To be honest, I don't know. @lliss and I were having trouble with this before when implementing the login stuff. I can try removing it from this view and see if I have any trouble.

It seems like we should never have to return it since we have the CSRFMiddleware app turned on.